src/clients/BundleDataClient/utils/FillUtils.ts

```diff
@@ -75,12 +66,26 @@ export async function verifyFillRepayment(
   _fill: FillWithBlock,
   destinationChainProvider: providers.Provider,
   matchedDeposit: DepositWithBlock,
-  possibleRepaymentChainIds: number[] = []
+  hubPoolClient: HubPoolClient
 ): Promise<FillWithBlock | undefined> {
   const fill = _.cloneDeep(_fill);
 
   const repaymentChainId = getRepaymentChainId(fill, matchedDeposit);
-  const validEvmRepayment = isEvmRepaymentValid(fill, repaymentChainId, possibleRepaymentChainIds);
+  const validEvmRepayment = isEvmRepaymentValid(fill, repaymentChainId);
+  if (!isSlowFill(fill)) {
```

Comment:
Should we be calling `verifyFillRepayment()` on slow fills?

Reply:
I think it depends where you want to make the `isSlowFill` check. Currently we call verifyFillRepayment before `isSlowFill` so this is best for reducing diff in the BundleDataClient for now